src/Uno.Foundation/Rect.cs

```diff
@@ -159,7 +159,7 @@ public Rect(Point point1, Point point2)
 		return sb.ToString();
 	}
 
-	public override string ToString() => (string)this;
+	public override string ToString() => IsEmpty ? "Empty." : base.ToString()!;
```

Comment:
Why is this change needed?

Reply:
Not needed. Just to match what I was seeing on WinUI

Reply:
Are you sure that non-empty rect would delegate to `base.ToString()` on WinUI?